common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -114,7 +115,15 @@ class LibraryContentBlock(
 
     show_in_read_only_mode = True
 
-    completion_mode = XBlockCompletionMode.AGGREGATOR
+    @property
```

Comment:
I wanted to use `classproperty` here to make the purpose clearer, but [it's not supported by pylint](https://github.com/PyCQA/pylint-django/issues/240) (also, [it doesn't seem to be documented before Django 3.1](https://docs.djangoproject.com/en/3.1/ref/utils/#django.utils.functional.classproperty), but [it's present in the code](https://github.com/django/django/blob/2.2.24/django/utils/decorators.py#L167-L176)).